conda_build/main_develop.py

```diff
@@ -42,6 +43,21 @@ def main():
                    help=("Relink compiled extension dependencies against "
                          "libraries found in current conda env. "
                          "Do not add source to conda.pth."))
+    p.add_argument(
+                   '-b', '--build_ext',
+                   action='store_true',
+                   help=("Build extensions inplace, invoking: "
+                         "python setup.py build_ext --inplace; "
+                         "add to conda.pth; relink runtime libraries to "
+                         "environment's lib/."))
+    p.add_argument(
+                   '-c', '--clean_build_ext',
```

Comment:
Looks like I need to delve a bit more into argparse so I can allow:
`conda develop --clean --build_ext pkg_path'

but not allow:
`conda develop --clean pkg_path`

Again, this is just me wanting to be explicit that clean in develop mode applies to build_ext and should not be confused with `conda clean`. I'll look into this and update